src/GeoPing.Api/Controllers/AccountController.cs

```diff
@@ -186,6 +152,21 @@ public IActionResult EditProfile(GeoPingUser user)
             return BadRequest(result);
         }
 
+        // GET /account/account/short
```

Comment:
What if user will send FullHD picture as avatar? would it be saved without resizing?

Reply:
It was supposed to picture would be send resized from front. If i`m wrong about it, we need tools, to upload that picture on the server filesystem  and resize it there. I don`t know any tools for that at this moment, so u r welcome for any tips and advices

Reply:
https://stackoverflow.com/questions/1922040/how-to-resize-an-image-c-sharp